internal/pkg/aws/sessions/sessions_test.go

```diff
@@ -171,6 +173,13 @@ func TestProvider_FromProfile(t *testing.T) {
 	})
 
 	t.Run("region information present", func(t *testing.T) {
+
+		ctrl := gomock.NewController(t)
+		defer ctrl.Finish()
+
+		m := sessionMocks.NewMocksessionValidator(ctrl)
+		m.EXPECT().ValidateCredentials(gomock.Any()).Return(credentials.Value{}, nil)
```

Comment:
with this new mock can we write a test case for 

`m.EXPECT().ValidateCredentials(gomock.Any()).Return(credentials.Value{}, context.DeadlineExceeded)` to see if we wrap the error appropriately?